lab-ben/README.md

```diff
@@ -0,0 +1,17 @@
+# Lab-08 Simple Data storage
```

Comment:
Good README, but try to be more explicit as if you're giving instructions to a user. I.e. how can I hit your API using curl commands? What do the endpoints return?

This will help a lot when you get into Project Week to write detailed docs.